### ISSUE TITLE
The "buffer" command has disappeared in Firefox 51.

### ISSUE THREAD

**Issue by cdown** [vimperator/vimperator-labs#698]
##### Issue type:

- Bug report

##### Version:

```
Vimperator: 3.15.0 (created: 2016/10/15 01:11:26)
Firefox:    Mozilla/5.0 (X11; Linux x86_64; rv:51.0) Gecko/20100101 Firefox/51.0
```

##### Description:

The `buffer` command has disappeared after upgrading from Firefox 50.1.0 to 51.0.1. Pressing `:` to open up the commands menu shows no comamnds called `buffer`, and trying to issue that command results in an error.

![2017-02-06_132956_790750525](https://cloud.githubusercontent.com/assets/660663/22648953/cfdc8646-ec70-11e6-9faf-4834ac6e82a8.png)

If I downgrade again, the `buffer` command reappears.

![2017-02-06_133113_292584068](https://cloud.githubusercontent.com/assets/660663/22648964/d77b3320-ec70-11e6-8fbb-cb0e13c6dfec.png)

Compare that with the output from 51.0.1:

![2017-02-06_133134_291907699](https://cloud.githubusercontent.com/assets/660663/22648970/dc07ef6e-ec70-11e6-977d-865adc88952c.png)

##### Expected behavior:

There is a `:buffer` command.

##### Steps to reproduce:

Upgrade to Firefox 51.0.1.

**Comment by cdown** [comment]
I notice that 3.16.0 is available on Github, but not addons.mozilla.org. Intentional?

I will go test with that since it seems it's more recent.

**Comment by HerringtonDarkholme** [comment]
3.1.5 is released the same day as on addon.mozilla.org. I wonder what's going wrong for 3.1.6.

**Comment by timss** [comment]
Thanks for reporting this issue, but this seems to be a duplicate of #568 and others, and was fixed in #675, ultimately released as [Vimperator 3.16](https://github.com/vimperator/vimperator-labs/releases/tag/vimperator-3.16.0). 

@cdown The new release, 3.16, has been submitted to addons.mozilla.org (AMO), but not yet approved. Not much else we can do at this point. 

Some more information/links for custom signed version etc., can be found here: https://github.com/vimperator/vimperator-labs/pull/675#issuecomment-276303232.
